users/models.py

```diff
@@ -143,3 +162,17 @@ def get_or_create_social_profile_from_slack_ids(slack_user_ids, slack_client=Non
         profiles[slack_user_id] = profile
 
     return profiles
+
+
+def build_email_verification_url(email, token):
+    """
+    Generates a url for the user to visit in the web app. If FRONT_END_HOST is not set, an exception will be raised.
+    :param email: A valid user email
+    :param token: Unique token to verify user email
+    """
+    base = settings.FRONT_END_HOST
+    query = {
+        'email': email,
+        'token': token,
+    }
+    return f'{base}/verify?{urlencode(query)}'
```

Comment:
@CoreyAR does this work for a URL to be directed to on the frontend? You can also just change it when you build out the functionality on the frontend.

Reply:
Works for me!